app/assets/javascripts/_practice_editor_header.es6

```diff
@@ -2,11 +2,19 @@
   function preventCloseModalFlickerOnPageLoad() {
     $(document).arrive("header", { existing: true }, () => {
       $("#dm-practice-editor-close-modal").find(".usa-modal--lg").removeClass("display-none");
+      $("#editing-guide-modal").find(".usa-modal--lg").removeClass("display-none");
```

Comment:
Do we need this line if we have a separate function that handles this (`preventEditingGuideModalFlickerOnPageLoad()`)?

Reply:
👀 Good catch! I've backed this out.